website/static/img/docusaurus.svg

```diff
@@ -1 +1,217 @@
872 284.440,335.600 284.000 C 335.328 283.560,334.497 283.202,333.753 283.205 C 332.914 283.208,333.296 283.677,334.759 284.440 M101.964 286.600 C 101.665 287.590,100.541 289.340,99.467 290.489 C 97.775 292.298,97.546 293.209,97.757 297.289 L 98.000 302.000 105.600 302.043 L 113.200 302.087 105.800 301.814 L 98.400 301.542 98.400 296.895 C 98.400 293.258,98.749 291.932,100.006 290.795 C 101.903 289.078,104.177 284.800,103.193 284.800 C 102.816 284.800,102.263 285.610,101.964 286.600 M341.498 286.671 C 342.102 287.402,343.251 288.000,344.051 288.000 C 344.851 288.000,345.728 288.360,346.000 288.800 C 346.272 289.240,347.643 289.551,349.047 289.491 C 351.167 289.401,351.261 289.327,349.600 289.058 C 347.603 288.734,345.024 287.757,342.000 286.178 C 340.569 285.431,340.516 285.484,341.498 286.671 M125.760 289.760 C 125.232 290.288,124.858 291.998,124.928 293.560 L 125.056 296.400 125.393 293.603 C 125.579 292.065,126.105 290.355,126.563 289.803 C 127.625 288.525,127.028 288.492,125.760 289.760 M355.137 290.612 C 357.765 291.953,358.085 293.402,355.846 293.829 L 354.000 294.182 355.825 294.291 C 356.829 294.351,357.864 293.843,358.125 293.162 C 358.643 291.813,356.367 289.660,354.400 289.637 C 353.740 289.630,354.072 290.068,355.137 290.612 M220.000 294.267 C 220.000 294.780,218.964 295.200,217.698 295.200 C 216.432 295.200,214.919 295.830,214.337 296.600 C 213.435 297.791,213.445 297.851,214.399 297.000 C 215.015 296.450,216.507 296.000,217.713 296.000 C 219.863 296.000,221.429 294.762,220.480 293.813 C 220.216 293.549,220.000 293.753,220.000 294.267 M348.574 296.493 C 348.303 296.764,346.893 297.267,345.441 297.611 C 343.144 298.155,343.067 298.247,344.847 298.318 C 345.973 298.363,347.593 297.871,348.447 297.225 C 349.301 296.579,349.790 296.039,349.533 296.025 C 349.277 296.011,348.845 296.222,348.574 296.493 M203.600 298.400 C 203.328 298.840,202.137 299.242,200.953 299.293 C 199.415 299.359,199.230 299.496,200.305 299.772 C 201.133 299.984,202.573 299.771,203.505 299.298 C 204.437 298.825,206.820 298.312,208.800 298.159 L 212.400 297.880 208.247 297.740 C 205.963 297.663,203.872 297.960,203.600 298.400 M122.400 300.667 C 122.400 301.180,121.410 301.649,120.200 301.709 C 118.181 301.809,118.155 301.847,119.892 302.177 C 121.839 302.546,123.920 301.253,122.947 300.280 C 122.646 299.979,122.400 300.153,122.400 300.667 M222.924 303.000 C 222.720 303.330,224.178 303.471,226.165 303.312 C 228.152 303.154,230.575 303.452,231.550 303.973 C 232.525 304.495,234.555 304.772,236.061 304.589 C 238.642 304.275,238.607 304.249,235.447 304.128 C 233.603 304.058,231.872 303.640,231.600 303.200 C 231.029 302.276,223.479 302.102,222.924 303.000 M219.993 305.836 C 217.446 309.753,216.685 311.491,217.994 310.405 C 218.657 309.854,219.200 309.027,219.200 308.565 C 219.200 308.104,219.798 307.065,220.528 306.258 C 221.259 305.451,221.695 304.629,221.498 304.431 C 221.301 304.234,220.623 304.866,219.993 305.836 M327.600 304.800 C 327.328 305.240,325.957 305.649,324.55
```

Comment:
what is going on with this file? is it a new svg?

Reply:
I can rename this.